pkgs/development/ocaml-modules/cow/default.nix

```
@@ -36,6 +36,7 @@ buildDunePackage rec {
       Markdown, to name but a few! This library provides OCaml combinators
       for these web formats.
     '';
+    homepage = "https://mirage.github.io/ocaml-cow/";
```

Maybe use
```suggestion
    homepage = "https://mirage.github.io/ocaml-cow/cow/Cow/index.html";
```

directly?